Docs/workflow.md

```diff
@@ -0,0 +1,41 @@
+# PROJECT DEVELOPMENT METHOD 
+
+## Repository and Dashboard
+The first step is create the repository where we are going to work on the project.
+
+Once we have our repository ready, the next step is to create a dashboard which is divided in 4 sections:
+ 1. To do list (All issues we have to accomplish)
+ 2. In progress (The issues we are currently working on)
+ 3. Code review (In this section are the issues that are resolved but are pending to be checked)
+ 4. Done (Here are the completed and checked issues)
+
+![Dashboard elemwave page project](/assets/docImages/dashboard.png)
+
+The next step is to generate the issues that we have to work on according to the requirements of the project and put them on the ToDo list. They must have an identifier to make easier the follow-up of the development.
+
+![ToDo List](/assets/docImages/ToDo.png)
+
+Then we have to assign the issues to the employees who are going to resolve them. After that we go to the second section. In  the "In progress" section, we have 
```

Comment:
Instead of "employees" it would be better to use "developers", also "team leader" or "project manager" to refer to the person in charge of reviewing and merging the code.